package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "frisby",
-  "version": "0.8.3",
-  "description": "Frisby.js: REST API Endpoint Testing built on Jasmine",
+  "name": "frisby-hack",
```

Comment:
Oh sorry - needed that in there for it to be able to be used. I'll create a new pull request, when I get some time, including some tests.